front/stat.graph.php

```diff
@@ -357,23 +357,23 @@
                                                    $_GET["type"], $val1, $val2);
 // Pass to hour values
 foreach ($values['avgsolved'] as $key => &$val) {
-   $val = round($val / HOUR_TIMESTAMP, 2);
```

Comment:
some greedy replacements here i think

Reply:
Yes, I knew I was too large on the replacements on two files, but this sounds simplier no to indoex those lines in commit... But I forgot :D It's fixed.